app/webpacker/controllers/link_controller.js

```diff
@@ -13,22 +13,14 @@ export default class extends Controller {
   }
 
   openExternalContentLinksInNewWindow() {
-    const links = this.contentTarget.querySelectorAll('a');
+    const links = Array.from(this.contentTarget.querySelectorAll('a'));
 
-    links.forEach((l) => {
-      if (this.isExternal(l)) {
+    links
+      .filter((l) => this.isExternal(l))
```

Comment:
We should probably modify links in either Ruby or JS, but not both; one for another day though 😬